### PR TITLE
Ansible Inventory Module - closes #288

### LINKED ISSUE #288: Ansible Inventory Module
- Configure Inventory File
- Create a playbook to ping the servers to make sure that the connection works

### PR DESCRIPTION
**Pull request by bob-crutchley**

### CODE REVIEW

File: topics/ansible/modules/inventory/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+The Ansible inventory is used for defining hosts in your infrastructure to connect to and configure.
+The default location for the inventory configurations are in the `/etc/ansible/hosts` file however, a different inventory file or hosts can be specified at any time when using the `-i` option on the command line:
+```bash
+# specify an inventory file
+ansible-playbook -i my_inventory playbook.yaml
+# specify a list of hosts
+ansible-playbook -i '192.168.1.123,192.168.1.124' playbook.yaml
+```
+
+## Configuration Overview
```
[the remark this answers is not in the document]
Comment (bob-crutchley) [reply topics/ansible/modules/inventory/README.md]: Not to sure what you mean sorry, do you mean putting the IP address in `/etc/hosts`?

File: topics/ansible/modules/inventory/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+# group of hosts
+[webservers]
+foo.example.com
+bar.example.com
+
+[dbservers]
+one.example.com
+two.example.com
+three.example.com
```
Comment (Matt25969) [inline topics/ansible/modules/inventory/README.md]:
```suggestion
three.example.com
```
http://35.176.33.104

File: topics/ansible/modules/inventory/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+```
```
Comment (Matt25969) [inline topics/ansible/modules/inventory/README.md]: Can we use Ip addresses here as well if they havent got a dns name set up yet? I thought I remember doing it but my Linux laptop has died

Comment (bob-crutchley) [reply topics/ansible/modules/inventory/README.md]: I've added something to clarify that you can use IP addresses in the Overview section and changed one of the host names in the example to an IP address.

**Pull request merged**